### ISSUE TITLE
Additional content for Selecting a Catalog in the editor

### ISSUE THREAD

**Issue by danid123** [GSA/openacr#340]
- [x] Change: "Select which catalog you will be using for the OpenACR." to "Select which catalog you want to use for this OpenACR."
- [x] Add helper text:
 - For VPAT® 2.4 Revised Section 508 Edition (WCAG 2.0): "This catalog is suggested if your product will be used only in the United States."
 - For VPAT® 2.4 WCAG 2.1 and Revised Section 508 Edition: "This catalog is suggested if your product will be used in the United States and in Europe."
 - For VPAT® 2.4 (WCAG 2.1) "This catalog is suggested if your product will only be used Europe."

**Comment by danid123** [comment]
@mgifford Is the above copy correct? Also is there any way to make the second option more similar to the other two? Can we just make it "VPAT® 2.4 Revised Section 508 Edition (WCAG 2.1)"?

**Comment by mgifford** [comment]
Ok, so https://gsa.github.io/openacr-editor/about

![Screen Shot of Catalog selection interface](https://user-images.githubusercontent.com/116832/156196151-8a65c963-c3ca-440a-8cc7-c0e62c391a7e.png)

The language is based on https://www.itic.org/policy/accessibility/vpat

Which uses:

- VPAT 2.4 508: [Revised Section 508 standards](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/final-rule/text-of-the-standards-and-guidelines) – the U.S. Federal accessibility standard
- VPAT 2.4 EU: [EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.01.01_60/en_301549v030101p.pdf) – the European Union’s “Accessibility requirements suitable for public procurement of ICT products and services in Europe”
- VPAT 2.4 WCAG: [WCAG 2.1](https://www.w3.org/TR/WCAG21) or ISO/IEC 40500 – W3C/WAI’s recently updated Web Content Accessibility Guidelines
- VPAT 2.4 INT: Incorporate all three of the above standards

I don't know that anyone outside of this team would have any real sense of what a Catalog is.  I also disagree with some of the helper text.  I've adjusted it below:

Maybe something like this:

## Select report type (and catalog)

Select which type of report you want to use for this OpenACR, this will select a corresponding machine-readable catalog. 

- VPAT® 2.4 Revised Section 508 Edition (WCAG 2.0)
    If you are are only concerned with meeting the [Revised 508 Standards and 255 Guidelines (January 22, 2018)](https://www.access-board.gov/ict/) or want to import an existing VPAT written to WCAG 2.0. 
- VPAT® 2.4 WCAG 2.1 and Revised Section 508 Edition
    If you want to align with both the Revised 508 Standards and the latest WCAG recommendations. This is a best practice for anyone who wants to build a future-compatible ACR in the USA. 
- VPAT® 2.4 (WCAG 2.1)
    If this is a web-based product or services that wants to align with the latest WCAG recommendations. This is a requirement for many organizations inside & outside of the USA. 

Future releases of OpenACR may build on VPAT 2.4 EU: [EN 301 549  v3.2.1] and VPAT 2.4 INT.

**Comment by danid123** [comment]
@mgifford I think it's too long. Think about it from the user's point of view. Helper text should be one sentence and task-specific.  Maybe it could be:
- This is the minimum standards and guidelines required in the United States of America.
- This is the best practice standard to be future-compatible in the United States of America.
- This is the international standard and is a higher standard accepted by many organizations in the United States of America.

But I thought the middle one was the one for both?

**Comment by mgifford** [comment]
What if we just used 6pt fonts? 

Ok....  How about this: 

- VPAT® 2.4 508: Revised Section 508 Edition (WCAG 2.0)
    These are the minimum standards required for Section 508 conformance.
- VPAT® 2.4 508 + WCAG: Revised Section 508 Edition (WCAG 2.1)
    This is the current best practice standard in the United States of America.
- VPAT® 2.4 WCAG: WCAG 2.1
    This is a web-specific ACR that follows the global best practices for web accessibility. 
    
Note: All this changes in June when WCAG 2.2 gets released. 

Afterthought: Maybe we should delete the WCAG 2.1 version and simply allow people to disable the Section 508 pages that aren't related to the web.  That would give you essentially the same results and would simplify the choices.

**Comment by danid123** [comment]
@mgifford I like the text edits. 

Disabling Section 508 would involve disabling on three pages? I think we should just leave it. I think it'd be too complex to explain in the Overview.

**Comment by danid123** [comment]
Following up on @dmundra 's comment from #338 :

> This may remove any entered criteria from your ACR that are not in the selected catalog. Download a copy of the report if you have not already. Are you sure that's what you'd like to do?

Maybe this would be better?

Switching catalogs may remove entered data and notes from your ACR that are not part of the newly selected catalog. Please download your report before switching catalogs to avoid losing information.

What do you think @mgifford @dmundra ?

**Comment by dmundra** [comment]
Ran into a bug #341 with the a radio button so added a confirm button to go with the change.
